File: CodingBat/Python/Warmup_2/front_times.py
#Given a string and a non-negative int n, we'll say that the front of the string is the first 3 chars, or whatever is there if the string is less than length 3. Return n copies of the front.
def front_times(str, n):
    if len(str)>=3:
        x=""
        y=str[:3]
        for i in range(n):
            x+=y
        return x
    else:
        x=""
        for i in range(n):
            x+=str
        return x

File: CodingBat/Java/Warmup_2/frontTimes.java
public class frontTimes{
    
    public static String frontTimes(String str, int n){
        // Given a string and a non-negative int n, we'll say that the front of the string is the first 3 chars, or whatever is there if the string is less than length 3. Return n copies of the front;

        String front="";
        if (str.length()<=3){
            front=str;
        }
        else{
            front=str.substring(0,3);
        }

        String str2="";
        for (int i=0;i<n;i++){
            str2+=front;
        }
        return str2;

    }
    
    public static void main(String[] args) {
        
    }
}